workitem/enum_type.go

```diff
@@ -56,6 +56,27 @@ func (t EnumType) Equal(u convert.Equaler) bool {
 	return true
 }
 
+// EqualEnclosing returns true if two EnumType objects are equal and/or the
+// values set is enclosing (larger and containing) the other values set.
+func (t EnumType) EqualEnclosing(u convert.Equaler) bool {
```

Comment:
Now that we use `Equal`, why do we need to repeat the field checks here again for SimpleType and BaseType? Can't we rename this function to just check the values? That would be much simpler.